internal/ethapi/api.go

```diff
@@ -1142,6 +1142,12 @@ func RPCMarshalHeader(head *types.Header) map[string]interface{} {
 		"transactionsRoot": head.TxHash,
 		"receiptsRoot":     head.ReceiptHash,
 	}
+
+	if head.BaseFee != nil {
+		result["baseFee"] = hexutil.Big(*head.BaseFee)
```

Comment:
```suggestion
		result["baseFee"] = (*hexutil.Big)(head.BaseFee)
```
I think it is better to follow the pattern above and do a type conversion.